frontend/public/components/utils/build-hooks-triggers.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import { K8sResourceKind } from '../../module/k8s';
+
+export const BuildHooksTriggers: React.SFC<BuildHooksTriggersProps> = ({ resource }) => {
+  const postCommitArgs = _.get(resource, 'spec.postCommit.args');
+  const postCommitCommand = _.get(resource, 'spec.postCommit.command');
+  const postCommitScript = _.get(resource, 'spec.postCommit.script');
+  const triggers = _.get(resource, 'spec.triggers');
+
+  return <React.Fragment>
+    {(postCommitArgs || postCommitCommand || postCommitScript) && <div className="co-m-pane__body">
```

Comment:
It feels like post commit hooks and triggers should be two separate components. Even though they have similar names, they're pretty different things.